src/index.html

```diff
@@ -11,7 +11,187 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
     <script type="text/javascript" src="./main.js"></script>
+    <form
+         action="http://localhost:3000/api"
+         method="post"
+         name="test-form"
+         >
```

Comment:
follow code styling [recommendations](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) point №3

Reply:
not fixed